src/BinSkim.Rules/DwarfRules/BA3003.EnableStackProtector.cs

```diff
@@ -171,8 +171,8 @@
                         context.Logger.Log(this,
                             RuleUtilities.BuildResult(FailureLevel.Error, context, null,
                                 nameof(RuleResources.BA3003_Error),
-                                context.TargetUri.GetFileName(),
-                                DwarfUtility.GetDistinctNames(failedList, context.TargetUri.GetFileName())));
+                                context.CurrentTarget.Uri.GetFileName(),
```

Comment:
>CurrentTarget [](http://example.com/codeflow?start=40&length=13)

This is the most intrusive change. The `TargetUri` property is now an enumerated artifact, i.e., a URI + contents (if loaded).